### PR TITLE
fix textures and more

### PR DESCRIPTION
**Pull request by agaertner**
Fixed the missing textures in newest bhud dev due to \\ instead of /
Fixed Smart Ping Menu CheckBox shown when ArcDps not available.
Quick Access Menu -> Smart Ping Menu
Removed `SendToChat() ` in favor of future `Chat.Send()` static from `GameService.GameIntegration.Chat` (https://github.com/blish-hud/Blish-HUD/pull/232)

(branch can be deleted after merge)

### CODE REVIEW

File: KillProof Module.csproj [hunk 1]
```diff
@@ -77,8 +77,7 @@
       <HintPath>packages\AsyncClipboardService.1.7.1\lib\net45\AsyncClipboardService.dll</HintPath>
     </Reference>
     <Reference Include="Blish HUD">
-      <HintPath>..\Blish-HUD\Blish HUD\bin\x64\Debug\Blish HUD.exe</HintPath>
-      <Private>False</Private>
```
Comment (dlamkins) [inline KillProof Module.csproj:81]: You need to double check that the Blish HUD exe is not set to copy local if this is not included.  Without this and if copy local is not set correctly, your module will package a copy of the entire Blish HUD exe within it which will make your module size huge.

File: manifest.json [hunk 2]
```diff
@@ -1,20 +1,24 @@
 {
   "name": "KillProof Module",
-  "version": "1.5.1",
-  "namespace": "Nekres.KillProof",
+  "version": "1.5.2",
+  "namespace": "KillProofModule",
```
Comment (dlamkins) [inline manifest.json:4]: Careful with changing the package namespace, FYI.

https://github.com/blish-hud/manifest.json/blob/master/manifest-v1.md#namespace

It resets any internal settings saved for your module for the user, if it's enabled, etc.

Comment (dlamkins) [reply manifest.json:4]: Are settings being used right now anyways?  If not, biggest thing it'll even do is just disable the module for anyone that already had it enabled.

File: KillProof Module.csproj (hunk 1 above)
Comment (agaertner) [reply KillProof Module.csproj:81]: https://github.com/blish-hud/KillProof-Module/pull/9/commits/5e7d4aed75cf40b2872c13f541203d361a94b854

File: manifest.json (hunk 2 above)
Comment (agaertner) [reply manifest.json:4]: Just if QuickMenu is enabled.

**Review by dlamkins — APPROVED** [review]

**Pull request merged**